packages/frontend/src/types/cms/pages/home.ts

```diff
@@ -7,7 +7,7 @@ export type HomePage = {
   meta_og?: StrapiComponent<MetaOpenGraph>;
   heading?: string;
   sub_heading?: string;
-  news_ticker?: StrapiComponent<Statement>[];
+  news_ticker?: { content: string };
```

Comment:
by the correct way:
this should be deleted both from FE and CMS

and General would have news_ticker

at the same time, if we combine General and HomePage in one query we could have one type CMS = StrapiSingleData<HomePage> & StrapiSingleData<General>


If you can - leave a comment something like: // todo: general CMS typescript types and create  global type CMS

Reply:
resolve in any case